scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -307,7 +303,11 @@ private String getBody() {
         } catch (UnknownHostException e) {
             logger.debug("Could not get hostname", e);
         }
-        return String.format(BODY_TEMPLATE, jobID, status, allTaskStatusesString, hostname);
+
+        final Properties properties = EmailConfiguration.getConfiguration().getProperties();
+        String templatePath = properties.getProperty(EmailConfiguration.TEMPLATE_PATH);
+        String bodyTemplate = new String(Files.readAllBytes(Paths.get(PASchedulerProperties.getAbsolutePath(templatePath))));
```

Comment:
Files.toString(file, Charset.defaultCharset()) seems better to me

Reply:
Done.